docs/core/tutorials/using-on-macos.md

```diff
@@ -180,74 +146,74 @@ namespace TestApp
     {
         [Fact]
         public void TestThing() {
-            Assert.Equal(42, new Thing().Get(19, 23));
+            Assert.NotEqual(42, new Thing().Get(19, 23));
         }
     }
 }
 ```
 
-Now, run `dotnet restore` and `dotnet build`. These commands will
-recursively find all projects to restore dependencies and build them.
-Finally, run `dotnet test test-library/test-library.csproj` to
-run the tests.
-The xUnit console test runner will run the one test, and report
-that it is passing. 
+Note that you assert the value 42 is not equal to 19+23 (or 42) when you first create the unit test (`Assert.NotEqual`), which will fail. An important step in building unit tests is to create the test to fail once first to confirm its logic.
+
+From the *golden* directory, execute the following commands:
+
+```console
+dotnet restore
+dotnet test test-library/test-library.csproj
+```
+
+These commands will recursively find all projects to restore dependencies, build them, and activate the xUnit test runner to run the tests. The single test fails, as you expect.
 
-### Writing the console app
+Edit the *UnitTest1.cs* file and change the assertion from `Assert.NotEqual` to `Assert.Equal`. Execute the following command from the *golden* directory to re-run the test, which passes this time:
 
-In your terminal run `dotnet new console -o app`
-to create a new console application. This project is also
-part of the solution, so run `dotnet sln add app/app.csproj`
-to add the project to the solution.
+```console
+dotnet test test-library/test-library.csproj
+```
 
-Your console application depends on the library you built and tested
-in the previous steps. You need to indicate that by running `dotnet add reference`
-again:
+## Create the console app
```

Comment:
I'd note here that the console app is going to call the library method just to make it clear what we're doing.